src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -376,8 +376,17 @@ bool IInteractiveWindowOperations.BreakLine()
         bool IInteractiveWindowOperations.Return()
         {
             return UIThread(uiOnly => uiOnly.Return());
+        }   
```

Comment:
Somehow Git (or whatever diff tool used by GitHub) decides this is the diff. But I did miss a newline before the next `#endregion`.